tests/Resources/Common.robot

```diff
@@ -156,6 +156,16 @@ Skip If RHODS Version Greater Or Equal Than
        Skip If    condition=${version-check}==True    msg=This test is skipped for RHODS ${version} or greater
     END
 
+Skip If RHODS Is Self-managed
+    [Documentation]    Skips test if RHODS is installed as Self-managed
+    [Arguments]    ${msg}=${EMPTY}
+    ${is_self_managed}=    Is RHODS Self-Managed
+    IF    "${msg}" != "${EMPTY}"
+       Skip If    condition=${is_self_managed}==True    msg=${msg}
+    ELSE
+       Skip If    condition=${is_self_managed}==True    msg=This test is skipped for Self-managed RHODS
```

Comment:
Is this needed? Can't you just pass an empty msg as a parameter?

Reply:
I copied this implementation from `Skip If RHODS Version Greater Or Equal Than` so I thought it was already analyzed as option, and I left as it is